packages/nimble-components/src/rich-text/models/tests/markdown-parser.spec.ts

```diff
@@ -21,6 +21,24 @@ describe('Markdown parser', () => {
         return nodes;
     }
 
+    function getEditorLastChildAttribute(
+        attribute: string,
+        doc: DocumentFragment
+    ): string {
+        return getEditorLastChildElement(doc)?.getAttribute(attribute) || '';
+    }
+
+    function getEditorLastChildElement(
+        doc: DocumentFragment
+    ): Element | null | undefined {
+        let lastElement = doc.lastElementChild;
+
+        while (lastElement?.lastElementChild) {
+            lastElement = lastElement?.lastElementChild;
+        }
+        return lastElement;
+    }
+
```

Comment:
For reviewer's info: Hoping this PR will take updates from [comment in another PR](https://github.com/ni/nimble/pull/1492#discussion_r1318953099) which directs to move these utility functions under `testing` folder

Reply:
I have moved these to `testing/markdown-parser-utils.ts` file